src/realm/group.hpp

```diff
@@ -741,11 +741,16 @@ class Group : private Table::Parent {
     ///
     ///   7 Introduced "history schema version" as 10th entry in top array.
     ///
+    ///   8 Subtables can now have search index.
+    ///
     /// IMPORTANT: When introducing a new file format version, be sure to review
     /// the file validity checks in Group::open() and SharedGroup::do_open, the file
     /// format selection logic in
     /// Group::get_target_file_format_version_for_session(), and the file format
     /// upgrade logic in Group::upgrade_file_format().
+
+    static constexpr int s_current_file_format_version = 8;
```

Comment:
I think `latest` or `writable` would be slightly better. `current` I think applies to the currently open file, which may be in read-only and thus only version 6 or 7. Maybe there's an even better name?